src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -54,6 +56,14 @@ public void update(StandaloneConfig standaloneConfig, List<String> ignoreGlobLis
             List<String> aliases = new ArrayList<>(author.getAuthorAliases());
             List<String> emails = new ArrayList<>(author.getEmails());
             aliases.add(author.getGitId());
+            allAliases.addAll(aliases);
+
+            HashSet<String> uniqueAliasesSet = new HashSet<>(allAliases);
+            boolean hasDuplicateAliases = uniqueAliasesSet.size() != allAliases.size();
+            if (hasDuplicateAliases) {
```

Comment:
So once the `allAliases` has a duplicate alias, it keeps logging this warning all the way to the end?

Reply:
I was confused whether we want something specific to show up or a general warning. So implemented it there in case we wanted something specific. Will update.